Python/instrumentation.c

```diff
@@ -1130,16 +1130,14 @@ _Py_Instrumentation_GetLine(PyCodeObject *code, int index)
 int
 _Py_call_instrumentation_line(PyThreadState *tstate, _PyInterpreterFrame* frame, _Py_CODEUNIT *instr, _Py_CODEUNIT *prev)
 {
+    // The caller should guarantee tstate->tracing == 0
```

Comment:
Either this should be an assert, or leave in the check for `tstate->tracing` below.

Reply:
Ah I was thinking "this should be an assertion" and then somehow I wrote a comment. It's a bit late yesterday :) Already fixed.